src/Psy/Command/WhereamiCommand.php

```diff
@@ -100,12 +100,12 @@ protected function fileInfo()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        extract($this->fileInfo());
+        $info = $this->fileInfo();
```

Comment:
list doesn't work unless fileInfo returns `[$file, $line]` instead of `compact('file', 'line')`